docs/modules/SDDs/pages/getting-started.adoc

```diff
@@ -0,0 +1,168 @@
+Create a namespace to host Lieutenant Operator and API
+[source,shell]
+----
+kubectl create namespace lieutenant
+----
+
+=== Install Lieutenant Operator
+
+Install *Lieutenant Operator* with the following commands:
+
+[source,shell]
+----
+# CRDs
+kubectl -n lieutenant apply -f https://raw.githubusercontent.com/projectsyn/lieutenant-operator/master/deploy/crds/syn.tools_clusters_crd.yaml
+kubectl -n lieutenant apply -f https://raw.githubusercontent.com/projectsyn/lieutenant-operator/master/deploy/crds/syn.tools_gitrepos_crd.yaml
+kubectl -n lieutenant apply -f https://raw.githubusercontent.com/projectsyn/lieutenant-operator/master/deploy/crds/syn.tools_tenants_crd.yaml
+
+# Deployment
+kubectl -n lieutenant apply -f https://github.com/projectsyn/lieutenant-operator/raw/master/deploy/service_account.yaml
+kubectl -n lieutenant apply -f https://github.com/projectsyn/lieutenant-operator/raw/master/deploy/role.yaml
+kubectl -n lieutenant apply -f https://github.com/projectsyn/lieutenant-operator/raw/master/deploy/role_binding.yaml
+kubectl -n lieutenant apply -f https://github.com/projectsyn/lieutenant-operator/raw/master/deploy/operator.yaml
+
+# Patch version
+kubectl -n lieutenant patch deployment lieutenant-operator -p '{"spec":{"template":{"spec":{"containers":[{"name":"lieutenant-operator","image":"docker.io/projectsyn/lieutenant-operator:v0.1.4"}]}}}}'
+----
+
+=== Install Lieutenant API
+
+Install *Lieutenant API* with the following commands:
+
+[source,shell]
+----
+kubectl -n lieutenant apply -f https://github.com/projectsyn/lieutenant-api/raw/master/deploy/service_account.yaml
+kubectl -n lieutenant apply -f https://github.com/projectsyn/lieutenant-api/raw/master/deploy/role.yaml
+kubectl -n lieutenant apply -f https://github.com/projectsyn/lieutenant-api/raw/master/deploy/role_binding.yaml
+kubectl -n lieutenant apply -f https://github.com/projectsyn/lieutenant-api/raw/master/deploy/deployment.yaml
+kubectl -n lieutenant apply -f https://github.com/projectsyn/lieutenant-api/raw/master/deploy/service.yaml
+
+kubectl -n lieutenant patch deployment lieutenant-api -p '{"spec":{"template":{"spec":{"containers":[{"name":"lieutenant-operator","image":"docker.io/projectsyn/lieutenant-api:v0.1.4"}]}}}}'
+----
+
+Check that the API is accessible:
+
+[source,shell]
+----
+kubectl -n lieutenant port-forward svc/lieutenant-api 8080:80 >/dev/null &
+curl localhost:8080/healthz
+----
+This should return `ok` as the answer to the `curl` command.
+
+TIP: The API documentation can be accessed in your browser under http://localhost:8080/docs while the kubectl port-forwarding is running.
+
+
+=== Configure Lieutenant Operator
+
+Create a secret which contains an access token to gitlab.com
+
+[source,shell]
+----
+kubectl -n lieutenant create secret generic vshn-gitlab --from-literal=endpoint=https://gitlab.com --from-literal=hostKeys="gitlab.com ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIAfuCHKVTjquxvt6CM6tdG4SLp1Btn/nOeHHE5UOzRdf" --from-literal=token=MYTOKEN
```

Comment:
It would probably reduce overall confusion if a link to https://syn.tools/lieutenant-api/deployment.html#_gitlab was added here, explaining why the secret must be named `vshn-gitlab`.

Reply:
Good input, added a few words about it.